src/routes/router.js

```diff
@@ -25,6 +26,10 @@ const Router = () => {
           path="/profile-builder"
           element={<ProtectedRoutes Component={Editor} />}
         />
+        <Route
```

Comment:
No need of any separate route for logout as its just a function.
after logout it should get redirected to login

Reply:
kk done